include/SDL3/SDL_filesystem.h

```diff
@@ -68,8 +68,8 @@ extern "C" {
  * The returned path is guaranteed to end with a path separator ('\\' on
  * Windows, '/' on most other platforms).
  *
- * The pointer returned is owned by the caller. Please call SDL_free() on the
- * pointer when done with it.
+ * The returned string follows the SDL_GetStringRule. This is different than
+ * SDL2! Do _not_ call SDL_free on the returned pointer!
```

Comment:
I don't think you need this warning here, we're returning const char *, which a compiler will warn about and won't let you free.

Reply:
Fixed in bbffc3a4940cddefcc85a4fcd6312ef8011bd296.